docs/sphinx/src/installation/installation.rst

```diff
@@ -8,50 +8,81 @@ Installation
 Building from Source
 ********************
 
-Create a build directory and navigate into this directory. Within this directory configure cmake:
+**Prerequisites**
 
-.. tip::
+* CMake >= 3.10
```

Comment:
@97gamjak shouldn't it be a higher version of cmake?

Reply:
Yes I think the dev branch needs now a newer version... But we have to check which version.

And then it would be best to set the the cake version in the ci to exactly the minimum version, so we see immediately if this changes

Reply:
So I tested it now with micromamba releases. So with cmake 3.13 it does not work - but the next cmake release I could test was 3.18 - so basically it would be best to just set it to 3.28 or? @Lucky-Armin @galjos

Reply:
Yes, I will adjust this with the next commit.